vnext/Desktop.IntegrationTests/DesktopTestRunner.cpp

```diff
@@ -72,7 +72,7 @@ shared_ptr<ITestInstance> TestRunner::GetInstance(
   devSettings->platformName = "windows";
 
   // Set to JSIEngineOverride::Chakra when testing the Chakra.dll JSI runtime.
```

Comment:
nit: should this comment still say Chakra?

Reply:
Probably no.